src/main/java/edu/usf/cutr/grha/model/ChicagoTncData.kt

```diff
@@ -38,4 +39,11 @@ data class ChicagoTncData
     @Parsed val pickupCentroidLongitude: Double = 0.0,
     @Parsed val dropoffCentroidLatitude: Double = 0.0,
     @Parsed val dropoffCentroidLongitude: Double = 0.0,
+    @Parsed var totalTravelTime: Int? = 0,
+    @Parsed var walkTime: Int? = 0,
+    @Parsed var transitTime: Int? = 0,
+    @Parsed var totalDistance: Double? = 0.0,
+    @Parsed var altitudeChange: Double? = 0.0,
+    @Parsed var transfers: Int? = 0,
+    @Parsed var Modes: List<TraverseModes>? = null
```

Comment:
Will a `List` flatten to a single CSV record correctly? I think it would need to look like `"BUS,WALK"`.

Reply:
Hmm... Good question. It would be easier to convert it to one `String `then, that way the `ChicagoTncData` class is not dependent on the library.